src/main/java/bpp/BeanPostProcessorImpl.java

```diff
@@ -0,0 +1,20 @@
+package bpp;
+
+import lombok.extern.log4j.Log4j;
+import org.springframework.beans.BeansException;
+import org.springframework.beans.factory.config.BeanPostProcessor;
+
+@Log4j
+public class BeanPostProcessorImpl implements BeanPostProcessor {
```

Comment:
Please explain, how do you use this class. I did not find its usage in the configuration file.